### PR TITLE
Update DHCP lease

### PR DESCRIPTION
**Pull request by cynepco3hahue**
It will also bring correct DNS server.

Signed-off-by: Lukianov Artyom <alukiano@redhat.com>

### CODE REVIEW

File: os-3.9-masteronly/scripts/node01.sh [hunk 1]
```diff
@@ -2,8 +2,12 @@
...
 
+# Wait until cluster will be up
 /usr/local/bin/oc get nodes
 while [ $? -ne 0 ]; do
     sleep 5
     /usr/local/bin/oc get nodes
 done
+
+# Update DHCP lease, will also update DNS servers
+dhclient
```
Comment (rmohr) [inline os-3.9-masteronly/scripts/node01.sh:13]: That is surprising. So far the nodes always had the right  ips, node names and dns servers after a fresh vm start. dnsmasq is already ready before the first vm starts. Could you explain why this is needed?

Comment (rmohr) [reply os-3.9-masteronly/scripts/node01.sh:13]: Oh, are the ansible-playbooks disabling dhcp?

Comment (cynepco3hahue) [reply os-3.9-masteronly/scripts/node01.sh:13]: I do not sure why it happens, I just saw that OpenShift VM had incorrect DNS name under `/etc/resolv.conf` and it prevented to pull images from registry container.

Comment (rmohr) [reply os-3.9-masteronly/scripts/node01.sh:13]: Ok, it concerns me a little, but if it works afterward as expected ...

**Pull request merged**